### DISCUSSION TITLE
deployment not working

### DISCUSSION THREAD

**Discussion by ugonnachidinmac** [vercel/community#7031] [Help]
### Summary

I deploy a project but it giving me error 
(404: NOT_FOUND
Code: NOT_FOUND
ID: cdg1::d6xsg-1717016602996-55c58dd50925)

### Example

youtubwork.vercel.app

### Steps to Reproduce

(404: NOT_FOUND
Code: NOT_FOUND
ID: cdg1::d6xsg-1717016602996-55c58dd50925)

**Comment by versecafe** [comment]
@ugonnachidinmac Can you share your repo or a reproduction of the issue?
